app/src/main/res/layout/activity_game_selection_page.xml

```diff
@@ -7,16 +7,20 @@
     tools:context="amazingme.activities.app.GameSelectionPage">
 
     <ImageView
```

Comment:
pretty sure we are gonna have to make this a bunch of containers on the page. 

because we are loading it dynamically, we need to be able to put each game's preview and button into the container and then populate the number of containers.... 

see if we can't consolidate this into the upper left corner or something.